src/main/mw.c

```diff
@@ -267,7 +251,7 @@ void annexCode(void)
             DISABLE_ARMING_FLAG(OK_TO_ARM);
         }
 
-        if (isCalibrating()) {
+        if (isCalibrating() || (averageWaitingTasks100 > 100)) {
```

Comment:
`isSystemOverloaded()` function?
technically it's not a CPU load - `averageSystemLoadPercent` ?